pom.xml

```diff
@@ -53,6 +53,7 @@
         <module>aws-sdk-java</module>
         <module>core</module>
         <module>services</module>
+        <module>services-hll/dynamodb-enhanced</module>
```

Comment:
Should we spread it out?  `services-high-level-libraries` I assume most people are not aware of the acronym

Reply:
Sure! I can move it once the next PR is merged. It felt kind of wordy the long way, and most customers wouldn't seen this, but hll is definitely very nebulous.